tools/lkl/lib/utils.c

```diff
@@ -205,3 +206,56 @@ void lkl_bug(const char *fmt, ...)
 
 	lkl_host_ops.panic();
 }
+
+int lkl_sysctl(char *path, char *value)
+{
+	int ret;
+	int fd;
+	char *delim, *p;
+	char full_path[256];
```

Comment:
Is it better to copy path into full_path first and then replace "." so that lkl_sysctl can be called with const char.